### ISSUE TITLE
coveralls reports 0% for python project

### ISSUE THREAD

**Issue by altaurog** [lemurheavy/coveralls-public#1492]
This might be a duplicate of #1256, #1329, or #1434, but since they are all different platforms I am opening a separate ticket.  My travis build ran several times for the same commit.  The first one appears in coveralls with the correct coverage, but the latest one (master) shows up with 0% coverage, although the coverage job succeeded and shows correctly in the jobs list.  Re-running the coverage job doesn’t seem to help.
https://coveralls.io/github/altaurog/pgcopy
https://coveralls.io/repos/267374/builds
https://coveralls.io/builds/34142357

**Comment by afinetooth** [comment]
Hi @altaurog,

I'm not sure why at the moment, but both of the most recent builds reporting 0% coverage (builds 91 & 92) have now updated themselves and show correct coverage---(93.38% coverage each):

https://coveralls.io/repos/267374/builds

![Screen Shot 2020-10-13 at 11 00 07 AM](https://user-images.githubusercontent.com/918667/95898009-42d29900-0d43-11eb-8c23-5436a33ddd9e.png)

I saw both builds change while looking at them.

Looking into why...

I did see a 422 error for Job 7 on Build 91 at Travis:
https://travis-ci.org/github/altaurog/pgcopy/jobs/735440414

![Screen Shot 2020-10-13 at 11 04 21 AM](https://user-images.githubusercontent.com/918667/95898413-d99f5580-0d43-11eb-8aa4-3064145499e2.png)

**Comment by afinetooth** [comment]
@altaurog I do notice [your `.travis.yml`](https://github.com/altaurog/pgcopy/blob/319d57cb1777cdae79a9d56f3cde978be60df47c/.travis.yml) has a significantly different pattern than [this example `.travis.yml`](https://github.com/coverallsapp/coveralls-node-demo/blob/master/.travis.yml) from our [sample project](https://github.com/coverallsapp/coveralls-node-demo) demonstrating how to configure Travis for parallel builds.

Specifically, you don't appear to be explicitly setting the `COVERALLS_PARALLEL=true` env var, or hitting our endpoint to close the parallel build with the notifications webhook:

```
 notifications:
    webhook: https://coveralls.io/webhook
```

Which of the [python language integrations for coveralls](https://docs.coveralls.io/python) are you using?

I'm afraid I'm not familiar enough with those, or with matrix builds at Travis, to know for sure if that difference is moot. Perhaps yours is set up correctly for your integration.

One thing we can do to learn a bit more about what you're sending to the API is to run in [verbose mode](https://docs.coveralls.io/troubleshooting#put-it-in-verbose--v).

I think you can probably pass the `-v` flag to where you're calling `coveralls` in your config. Otherwise, the README for your integration may specify setting an env var like `COVERALLS_VERBOSE=true`, etc.

Can you provide more explanation on how / why you set up your `.travis.yml` the way you did, and please try another build in verbose mode? (After that, we can examine the Travis build log together for more clues.)

**Comment by altaurog** [comment]
Hi, @afinetooth thanks so much for the quick response.  It looks like I am using [coveralls-python](https://github.com/coveralls-clients/coveralls-python).  I honestly have no idea why I did it that way.  I added coveralls over a year ago, but it looks like parallel builds were added to the example .travis.yml only about 6 months ago.  I must have been certain it was enough to collect coverage stats from one run, but if so, it does seem strange that coveralls would run after them all.  I will try to get verbose output for further diagnostic.

**Comment by altaurog** [comment]
I updated my config a bit.  For now will just run coveralls in one job.  Also I reviewed the documentation for the integration and saw that I need to expose some environment variables from travis.  Not sure if that is what solved the problem, but there are no exceptions in the coveralls report now.  Thanks again for the help, will come back if I get stuck again.
https://travis-ci.org/github/altaurog/pgcopy/jobs/735737317
https://coveralls.io/builds/34169757
